Documentation/gettingstarted/hubble-install.rst

```diff
@@ -0,0 +1,40 @@
+.. tabs::
+
+   .. group-tab:: Linux
+
+      Download the latest hubble release:
+
+      .. parsed-literal::
+
+         curl -L https://github.com/cilium/hubble/releases/latest/download/hubble-linux-amd64.tar.gz | tar zxf -
```

Comment:
Don't you want to pin a specific version instead of using latest? (at least in the Cilium's stable branches)

Reply:
I think this is done on purpose: https://github.com/cilium/hubble/issues/292

Reply:
> @aanm @Rolinh i was a bit hesitant to pin a version in the cilium documentation. i felt like that would add too much overhead for maintaining the documentation, and it's easy to forget to keep it up to date.
> 
> how about having `stable.txt` like how kubectl does it: https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-linux this way the documentation doesn't have to be updated, and we can control which version is considered "stable" from hubble release page.

This is a great idea!

Reply:
https://github.com/cilium/hubble/issues/298